[24.2 release docs] Add Gobal Nextflow config field to compute environment pages (#166)
seqeralabs/docs: platform_versioned_docs/version-24.1/compute-envs/aws-batch.mdx

24. Select **Dispose resources** to automatically delete these AWS resources if you delete the compute environment in Seqera Platform.
25. Apply [**Resource labels**](../resource-labels/overview.mdx) to the cloud resources consumed by this compute environment. Workspace default resource labels are prefilled.
26. Expand **Staging options** to include:
  - Optional [pre- or post-run Bash scripts](../launch/advanced.mdx#pre--post-run-scripts) that execute before or after the Nextflow pipeline execution in your environment.

>   - Global Nextflow configuration settings for all pipelines launched with this environment. Configuration values in this field override the same values in the pipeline Nextflow config file. 

Reviewer: ```suggestion
  - Global Nextflow configuration settings for all pipelines launched with this environment. These configuration values will override the matching values in the pipeline Nextflow config file. 
```

**Create a manual Seqera compute environment**

1. In a workspace, select **Compute environments > New environment**.
1. Enter a descriptive name for this environment, e.g., _AWS Batch Manual (eu-west-1)_.
1. Select **AWS Batch** as the target platform.
1. Select **+** to add new credentials.
1. Enter a name for the credentials, e.g., _AWS Credentials_.
1. Enter the **Access key** and **Secret key** for your IAM user.

    :::note
    You can create multiple credentials in your Seqera environment. See [Credentials](../credentials/overview.mdx).
    :::

1. Select a **Region**, e.g., _eu-west-1 - Europe (Ireland)_.

> 1. Enter an S3 bucket path for the **Pipeline work directory**, e.g., `s3://seqera-bucket`. This bucket must be in the same region chosen in the previous step.

Reviewer: ```suggestion
1. Enter an S3 bucket path for the **Pipeline work directory**, e.g., `s3://seqera-bucket`. This bucket must be in the same region you chose in the previous step.
```

    :::

1. Set the **Config mode** to **Manual**.
1. Enter the **Head queue**, which is the name of the AWS Batch queue that the Nextflow main job will run.
1. Enter the **Compute queue**, which is the name of the AWS Batch queue where tasks will be submitted.
1. Apply [**Resource labels**](../resource-labels/overview.mdx) to the cloud resources consumed by this compute environment. Workspace default resource labels are prefilled.
1. Expand **Staging options** to include:
    - Optional [pre- or post-run Bash scripts](../launch/advanced.mdx#pre--post-run-scripts) that execute before or after the Nextflow pipeline execution in your environment.

>     - Global Nextflow configuration settings for all pipelines launched with this environment. Configuration values in this field override the same values in the pipeline Nextflow config file. 

Reviewer: ```suggestion
    - Global Nextflow configuration settings for all pipelines launched with this environment. These configuration values will override the matching values in the pipeline Nextflow config file.
```
